GOVERNANCE.md

```diff
@@ -36,10 +36,17 @@ Collaborators. All pull requests must be reviewed and accepted by a
 Collaborator with sufficient expertise who is able to take full
 responsibility for the change. In the case of pull requests proposed
 by an existing Collaborator, an additional Collaborator is required
-for sign-off. Consensus should be sought if additional Collaborators
-participate and there is disagreement around a particular
-modification. See [Consensus Seeking Process](#consensus-seeking-process) below
-for further detail on the consensus model used for governance.
+for sign-off.
+
+If there is disagreement among Collaborators about whether a proposed change
+should be accepted, then the change may not be accepted unless:
```

Comment:
Does this sound okay? I am finding it very difficult to understand the double negatives and the meaning of this sentence